chia-bls/src/gtelement.rs

```diff
@@ -153,3 +91,65 @@ impl Streamable for GTElement {
         ))
     }
 }
+
+#[cfg(feature = "py-bindings")]
+mod py_bindings {
+    use super::*;
+
+    use chia_traits::{from_json_dict::FromJsonDict, to_json_dict::ToJsonDict};
+    use pyo3::{exceptions::PyValueError, prelude::*};
+
+    #[pymethods]
+    impl GTElement {
```

Comment:
I didn't know this worked. by I suppose since you `use super::*;`, this `impl` block refers to the `GTElement` of the outer scope?

Reply:
Yes, you can add an impl block in a separate module. It also affects visibility, making it relative to the module you're implementing it in, which can be useful in some cases.